src/app/pages/server-compare/server-compare.component.ts

```diff
@@ -216,12 +225,18 @@ export class ServerCompareComponent implements OnInit, AfterViewInit {
     private route: ActivatedRoute) { }
 
   ngOnInit() {
+    const id = this.route.snapshot.queryParamMap.get('id');
+
+    if(id) {
+      const specialCompare = this.specialCompares.find((x: any) => x.id === id);
+      if(specialCompare) {
+        this.title = specialCompare.title;
+        this.description = specialCompare.description;
+        this.breadcrumbs.push({ name: specialCompare.title, url: `/compare`, queryParams: { id: specialCompare.id } });
```

Comment:
maybe also show an extra breadcrumb item for the `instances` type of comparison, e.g. adding "3 selected servers"?

![image](https://github.com/user-attachments/assets/37e322bb-e206-4181-9aa3-1c91e00b3576)